src/main/java/com/quantumsoft/qupathcloud/entities/Datasets.java

```diff
@@ -18,15 +18,32 @@
 import java.util.ArrayList;
 import java.util.List;
 
+/**
+ * The Datasets class contains a list of {@link com.quantumsoft.qupathcloud.entities.Dataset}
```

Comment:
Datasets is the list of {@link com.quantumsoft.qupathcloud.entities.Dataset} retrieved from the server. ?